test/libsyncengine/reconciliation/platform_inconsistency_checker/testplatforminconsistencycheckerworker.cpp

```diff
@@ -154,6 +165,78 @@ void TestPlatformInconsistencyCheckerWorker::testNameClash() {
 #endif
 }
 
+void TestPlatformInconsistencyCheckerWorker::testNameClashAfterRename() {
+    // Create local files
+    std::vector<std::string> nodes = {{"a1"}, {"A"}};
+    for (const auto &n : nodes) {
+        std::ofstream ofs(_tempDir.path / n);
+        ofs << "Some content.\n";
+        ofs.close();
+    }
+
+    // Set up DB
+    DbNode dbNode_A(2, _syncPal->_syncDb->rootNode().nodeId(), Str("A"), Str("A"), "lA", "rA", defaultTime, defaultTime,
```

Comment:
Not sure if it is merged yet, but there might be a `const` public accessor to `_syncDb`.